code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -244,7 +244,7 @@ emp_act
 	apply_damage(I.force, I.damtype, affecting, armor, sharp=weapon_sharp, edge=weapon_edge, used_weapon=I)
 
 	var/bloody = 0
-	if(((I.damtype == BRUTE) || (I.damtype == HALLOSS)) && prob(25 + (I.force * 2)))
+	if(I.damtype == BRUTE)

```

Comment:
er, you removed something important here, weapons will now always get blood on them

the conditional read like
```
if damtype == BRUTE OR damtype == HALLOSS
    if prob(25 + (I.force *2))
```
not
```
if damtype == BRUTE
    etc
if damtype == HALLOSS AND prob(25 + (I.force * 2))
    goto above